pkgs/servers/jellyfin/web.nix

```
@@ -29,7 +29,7 @@ buildNpmPackage rec {
     runHook postInstall
   '';
 
-  passthru.updateScript = nix-update-script {};
+  passthru.updateScript = nix-update-script { };
```

```suggestion
  passthru.updateScript = nix-update-script {};
```

The formatting change here doesn't seem to come from master?

---

Looks like the bot can do the backport: #271075